saleor/tests/e2e/checkout/discounts/promotions/test_promotion_applied_on_checkout_with_specific_total.py

```diff
@@ -0,0 +1,141 @@
+import pytest
+
+from ....product.utils.preparing_product import prepare_product
+from ....promotions.utils import create_promotion, create_promotion_rule
+from ....shop.utils.preparing_shop import prepare_default_shop
+from ....utils import assign_permissions
+from ...utils import (
+    checkout_complete,
+    checkout_create,
+    checkout_delivery_method_update,
+    checkout_dummy_payment_create,
+    checkout_lines_update,
+)
+
+
+def prepare_promotion(e2e_staff_api_client):
+    shop_data = prepare_default_shop(e2e_staff_api_client)
+    channel_id = shop_data["channel"]["id"]
+    channel_slug = shop_data["channel"]["slug"]
+    warehouse_id = shop_data["warehouse"]["id"]
+    shipping_method_id = shop_data["shipping_method"]["id"]
+
+    promotion_name = "Promotion"
+
+    promotion_data = create_promotion(e2e_staff_api_client, promotion_name)
+    promotion_id = promotion_data["id"]
+    discount_value = 15
+    order_predicate_total_value = "15"
+    order_predicate = {
+        "discountedObjectPredicate": {
+            "baseTotalPrice": {"eq": order_predicate_total_value}
+        }
+    }
+
+    input = {
+        "promotion": promotion_id,
+        "channels": [channel_id],
+        "name": "test rule",
+        "orderPredicate": order_predicate,
+        "rewardType": "TOTAL_DISCOUNT",
```

Comment:
We are getting rid of `TOTAL_DISCOUNT` for now (https://github.com/saleor/saleor/pull/15229)